src/activities/discussions/DiscussionTopicEntity.js

```diff
@@ -69,6 +69,27 @@ export class DiscussionTopicEntity extends Entity {
 		return fields && fields.value;
 	}
 
+	/**
+	 * @returns {number} participation option of discussion topic.
+	 */
+	participationSelection() {
+		const options = this.participationOptions();
+		const selected = options?.find(option => option?.selected);
+		return selected?.value;
```

Comment:
Should we have a default selection or is it ok to just have null here?

Reply:
I think it should be fine - paging doesn't use a default either: https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/master/src/activities/ActivityUsageCollectionEntity.js#L75